kernel/src/table_changes/log_replay/tests.rs

```diff
@@ -523,18 +524,22 @@ async fn data_skipping_filter() {
         column_expr!("id"),
         Scalar::from(4),
     );
+    let logical_schema = get_schema();
+    let predicate = match PhysicalPredicate::try_new(&predicate, &logical_schema) {
+        Ok(PhysicalPredicate::Some(p, s)) => Some((p, s)),
+        other => panic!("Unexpected result: {:?}", other),
```

Comment:
Why do we panic here? seems that `try_new` could return `None` or `StaticSkipAll`

Reply:
nvm it's for test code